keyboards/protokeeb/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "keyboard_name": "Protokeeb",
+    "manufacturer": "A-Tech Officials",
+    "maintainer": "atechofficials",
+    "url": "https://github.com/atechofficials/protokeeb",
+    "bootloader_instructions": "Press and hold the FN Key (row-4, column-1) then press the key mapped to `QK_BOOT` i.e ENTER Key (row-4, column-4) to enter the bootloader mode.",
```

Comment:
`bootloader_instructions` should only be hardware related, and not instructions on bootmagic or the reset keycode. 
```suggestion
```

Reply:
Ok understood.